src/data-table/_data-table.scss

```diff
@@ -24,6 +24,7 @@
   white-space: nowrap;
   font-size: 13px;
   background-color: unquote("rgb(#{$color-white})");
+  -webkit-transform: translate3d(0,0,0);
```

Comment:
A comment should be added then so it is clear as to why this one is prefixed.

Ideally though, translation would net a perf benefit in any browser that uses it. So not-prefixing it would be ideal until something happens that shows translation is causing a problem elsewhere.